firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# Unreleased (21.3.1)
+# Unreleased (21.3.2)
+- [changed] Transactions previously required that every document read in a
+  transaction must also be written to. This requirement has been removed, such
```

Comment:
"Written to" on the end of the sentence is a little clunky. Consider "that every document read in a transaction must also be written".